package.json

```diff
@@ -46,6 +46,7 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-prettier": "^4.0.0",
+    "husky": "^6.0.0",
```

Comment:
should there also be a `postinstall` to initialize git hooks automatically, or does that have to be a manual step after npm install?

Reply:
Hi @joshuali925 , good catch! yeah you are right, if there is a `postinstall` script in package.json, then people don't need to run `yarn husky install` manually, let me add it in a fast-follow. Thanks!